crux-lucene/test/crux/lucene_test.clj

```diff
@@ -168,7 +177,8 @@
     (submit+await-tx [[:crux.tx/put {:crux.db/id "ivan" :name "Ivan"}]])
     (submit+await-tx [[:crux.tx/put {:crux.db/id "ivan" :name "Ivan"}]])
 
-    (t/is (= 2 (l/doc-count)))
+    (t/is (= 2 (doc-count (-> @(get-in *api* [:query-engine :!pred-ctx])
```

Comment:
if we keep doing this access pattern, could establish an fn in the lucene fixtures ns